source/blog/2019-01-11-the-ember-times-issue-80.md

```diff
@@ -12,8 +12,10 @@ Grüezi mitenand Emberistas! 🐹
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [DecEmber is over? Collect your gift!](https://airtable.com/shrm9o5W89wsAa1Tq)
+[DecEmber](https://www.emberjs.com/blog/2018/11/29/december-event.html) has wrapped up and it seems to have bit a big hit! There were many contributions to `ember-learning` repos and it is all thanks to the awesome Ember community! 
 
+Did you participate in **DecEmber**? You are eligable for a nice gift! Just fill out the form and enjoy!
```

Comment:
I think it would be really cool to link to the form here another time:

```suggestion
Did you participate in **DecEmber**? You are eligible for a nice gift! Just [fill out the form](https://airtable.com/shrm9o5W89wsAa1Tq) and enjoy!
```